tests/common.py

```diff
@@ -76,6 +73,9 @@ def stop_hass():
     hass.start = start_hass
     hass.stop = stop_hass
 
+    # FIXME should not be a daemon. Means hass.stop() not called in teardown
+    threading.Thread(name="LoopThread", target=run_loop, daemon=True).start()
```

Comment:
I suppose moving the FIXME comment around should have been a clue to give it a try. :) It's working on my system. Pushing it here for travis. I know @balloob did some fixing of test instance cleanup lately, so that might have resolved the issue here.